### PR TITLE
guarantee odds by matching competitor ID

### PR DESCRIPTION
**Pull request by edgethio**
Guarantee that outcome maps to the correct competitor

### CODE REVIEW

File: src/components/SelectEvent.js [hunk 1]
```diff
@@ -176,7 +173,7 @@ const SportSection = ({section, eventSelected}) => {
     )
 }
 
-const OutcomesRow = ({event, wagerMembers, selectedOutcome, selectedMarket, eventSelected, rowNum}) => {
+const OutcomesRow = ({event, wagerMembers, selectedOutcome, selectedMarket, eventSelected, competitorId}) => {
     const auth = useStoreState(state => state.firebase.auth);
     if (!!wagerMembers) {
         const selected = wagerMembers[rowNum]?.uid === auth.uid;
```
Comment (edgethio) [inline src/components/SelectEvent.js]: @donniemattingly , how can rowNum refer both to the "wagerMembers" and the outcome?

Comment (donniemattingly) [reply src/components/SelectEvent.js]: It's from src/components/PersonalWagers.js:236

```javascript

const membersFromWager = (wager) => {
    return {
        [wager.details.outcome]: wager.proposedBy,
        [(wager.details.outcome + 1) % 2]: wager.proposedTo
    }
}

```

Basically when I save a wager I saved the outcome selected by the person proposing it, and assume the 'other' outcome must be the person being proposed to.

Comment (edgethio) [reply src/components/SelectEvent.js]: so this is only for displaying saved wagers?

Comment (donniemattingly) [reply src/components/SelectEvent.js]: yeah, that's correct - also for 'confirming' wagers, but that screen doesn't show anymore

**Review by donniemattingly — APPROVED** [review]

**Pull request merged**